amp_conf/htdocs/admin/libraries/Console/Chown.class.php

```diff
@@ -149,6 +149,9 @@ protected function execute(InputInterface $input, OutputInterface $output, $quie
 			$this->modfiles['framework'][] = array('type' => 'execdir',
 			'path' => $AMPBIN,
 			'perms' => 0755);
+			$this->modfiles['framework'][] = array('type' => 'dir',
```

Comment:
i am hesitant about this because it could set sudo binaries to user asterisk:asterisk. Need to have rob comment on this.